openstack/big_patch.py

```diff
@@ -668,6 +668,22 @@ def deploy_to_node(self, node, nodes_information):
         self.check_lldpd_running(node)
         self.check_bond_int_speeds_match(node, bond_interfaces)
 
+        # copy neutron metadata file to all nodes
+        resp, errors = self.env.copy_file_to_node(node, "/etc/neutron/metadata_agent.ini",
+                                                  "/etc/neutron/metadata_agent.ini")
+        if errors:
+            raise Exception("error pushing updated metadata.ini to %s:\n%s"
```

Comment:
ok. also since the next step is dependent on this one, I'll instead work as:
```py
# copy metadata file
self.env.copy_file_to_node
if not errors:
    # restart agents
    self.env.run_cmd_on_node
    if errors:
        # print restart unsuccessful in logs
else:
    # print copy unsuccessful in logs
```